### ISSUE TITLE
Alex not discovering new devices

### ISSUE THREAD

**Issue by Fitz2380** [bwssytems/ha-bridge#1233]
Alexa will not discover new devices.  I use ha-bridge with several devices using the Vera Plus hub.  I recently had to replace one of these devices due to failure (zwave switch) and I cannot get Alexa to discover the new device.  I have gone into ha-bridge and added the device and it appears in the bridge device list.  I can turn the device off and on using the ha-bridge controls. 

I was running 5.3.1 and have tried teh 5.3.1RC1 with no change.  I have deleted the device from ha-bridge and re-added it.   I have tried discovery with several of my echos ( I have 5 or so different vintage).  I have used the Amazon app to rediscover a device.  No change.  

I have read through the other posted issues that had similar problems, but it does not seem that any of these ever have a resolution that works for all. 

I do not want to delete all devices and re-add because the rest are working well and I do not want to take the risk that no device will be re-added.  

I am using port 80, I did add a new Vera scene about 2 or 3 weeks ago and that worked.  

I do have UPNP tracing turned on and attached the debug log.  I asked Alexa to discover around 12:42:30 and it said complete around 12:43:17 with no new devices. 

[habridge debug.txt](https://github.com/bwssytems/ha-bridge/files/5210782/habridge.debug.txt)

**Comment by Fitz2380** [comment]
Additional info:  192.168.0.118 is a harmony hub; 192.168.0.10 is the ha-bridge server (windows 10).

**Comment by Fitz2380** [comment]
Update:  I was able to get everything working again.  After much digging into related issues and much troubleshooting, Amazon has now recognized my new devices.  What resolved the issues?  Here are the steps that I took that I believe made it work: 

Using the Amazon app (or web) devices page, I deleted duplicate devices (some labeled as Phillips Royal) and the duplicate entry   identified as (offline).  I deleted all of the offline duplicates which resulted in the deletion of all of the devices.  

Attempting to perform Amazon discovery at this point still did not work.   I now had my worst fear with all of my ha-bridge devices missing... 

I then renumbered my devices in ha-bridge and changed the starting index from 200 to 300.  I now had new numbered ha-bridge devices.  I then performed a Amazon discovery and all of my devices were found!!!!.  I even renamed a device in ha-bridge and a Amazon discovery found the new name for the device.

I am still perplexed as to why this makes it work.  It must have something to do with the index numbers and if there is some conflict due to duplicates that is not obvious from web device view or the App device view. 

Anyways my wife is happy once again.  I greatly appreciate the time and effort that goes into this product and am willing to help support this in the future with anything I can do.  I am retiring soon (EE) and love  to poke around in programming, networking,etc and will have plenty of time to do so.
